src/VisualStudio.Roslyn.SDK/ComponentDebugger/Roslyn.ComponentDebugger.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <RootNamespace>Roslyn.ComponentDebugger</RootNamespace>
+    <TargetFramework>net472</TargetFramework>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Design" />
+    <Reference Include="System.Drawing" />
+    <Reference Include="System.Windows.Forms" />
+    <Reference Include="System.Xml" />
+    <Reference Include="WindowsBase" />
+    <Reference Include="PresentationCore" />
+    <Reference Include="PresentationFramework" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudio.ProjectSystem.Managed" Version="16.8.1-beta1-928-02" PrivateAssets="all" />
```

Comment:
⁉ These packages don't seem to be on a public feed anywhere? It works when you add one of the devdiv feeds, but they are not public. I tried to get it working with earlier packages but kept hitting NuGet versioning hell. We'll need to fix this for release, but I don't have a great idea how to.

Reply:
@dmonroym what it the correct way to get the latest packages?

Reply:
It's published in https://dev.azure.com/azure-public/vside/_packaging?_a=feed&feed=vs-impl

Reply:
thanks @chsienki this feed is public, we should just add it to the nuget.config for this repo